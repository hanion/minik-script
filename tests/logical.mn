// logical

print false and 2; // false
print true  and 2; // 2
print false or 2;  // 2
print true  or 2;  // true
print false or true and "this"; // "this"
