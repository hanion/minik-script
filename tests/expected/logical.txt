false
2.000000
2.000000
true
this
